.github/workflows/development.yml

```diff
@@ -21,4 +21,16 @@ jobs:
         ruby-version: ${{ matrix.ruby }}
         bundler-cache: true
 
-    - run: bundle exec rake
+    - run: bundle exec rake spec
+
+  lint:
```

Comment:
I would probably put this as a separate workflow.

Reply:
Like separate file?

Reply:
Yeah like `lint.yml`, `rubocop.yml` or something similar?

Reply:
I don't see any benefit in there. Is that blocker?

Reply:
I personally prefer it just because I think it's easier to share between repos.

Reply:
I can split it, that is no problem. Just ping me if that's needed.